src/main/webapp/index.html

```diff
@@ -2,13 +2,13 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>Index</title>
+    <title>Main Hub</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="navigation">
-      <a href="main_hub.html" id="hub">Main Hub</a>
+      <a href="index.html" id="hub">Main Hub</a>
```

Comment:
I was actually thinking the index.html page would be the page that directs the user to log in. I already made it, unless you feel strongly about making it tied to the main hub

Reply:
You're way's probably better. However, does the login page block any viewers from visiting the website on "guest mode", or should we allow people to view the website even if they're not logged in?

Reply:
I think if they're not logged in, their projects, tasks, checklist, etc are all empty

Reply:
The way I imagine it, people would only have access to the website if logged in

Reply:
> I was actually thinking the index.html page would be the page that directs the user to log in

If user was already logged in, would they be sent back to the login page again if they go back to the home (index) page?

There's also the question of users going directly to "site.com/task.jsp?12345".  Assuming the task ID is correct, how should this behave if they were logged in vs. if they were not logged in?